docs/site/Where-filter.md

```diff
@@ -0,0 +1,611 @@
+---
+title: 'Where filter'
+lang: en
+keywords: LoopBack 4.0, LoopBack 4, where
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Where-filter.html
+summary: A <i>where</i> filter specifies a set of logical conditions to match, similar to a WHERE clause in a SQL query.
+---
+
+## REST API
+
+In the first form below, the condition is equivalence, that is, it tests whether _property_ equals _value_.
+The second form below is for all other conditions.
+
+```
+filter[where][property]=value
+```
+
+```
+filter[where][property][op]=value
+```
+
+Where:
+
+- _property_ is the name of a property (field) in the model being queried.
+- _value_ is a literal value. 
+- _op_ is one of the [operators](#operators) listed below.
+
+For example, if there is a `Car` model with an `odo` property, the following query finds instances where the `odo` is greater than 5000:
+
+```
+/cars?filter[where][odo][gt]=5000
+```
+
+For example, here is a query to find cars with `odo` is less than 30,000:
+
+```
+/cars?filter[where][odo][lt]=30000
+```
+
+You can also use [stringified JSON format](Querying-data.html#using-stringified-json-in-rest-queries) in a REST query.
```

Comment:
Let's show a example of URL with stringified JSON format for the `where` filter.

Reply:
Added a couple of examples!